torchtune/utils/multi_dataset.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List, Tuple
+
+from torch.utils.data import Dataset
+
+from torchtune import utils
+
+log = utils.get_logger("DEBUG")
+
+
+class MultiDataset(Dataset):
+    def __init__(self, datasets: List[List[Dataset]]):
```

Comment:
Why List[List[ here?

Reply:
Hi, oops, it's a typo from the first iterations of the class, changed to `List[Dataset]` as it should be, thanks!